examples/java/web-service/src/main/java/com/mycompany/app/WebService.java

```diff
@@ -0,0 +1,103 @@
+package com.mycompany.app;
+
+import software.constructs.Construct;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.cdk8s.App;
+import org.cdk8s.Chart;
+import org.cdk8s.ChartOptions;
+
+import k8s.IntOrString;
+import k8s.LabelSelector;
+import k8s.ObjectMeta;
+import k8s.PodTemplateSpec;
+import k8s.Service;
+import k8s.ServiceOptions;
+import k8s.ServicePort;
+import k8s.ServiceSpec;
+import k8s.DeploymentSpec;
+import k8s.PodSpec;
+import k8s.Container;
+import k8s.ContainerPort;
+import k8s.Deployment;
+import k8s.DeploymentOptions;
```

Comment:
I expected these to be `import imports.k8s.Service`... Pom.xml file still feels too complex.

Reply:
Let me see if I can cut some stuff out now with the changes to `jsii-srcmak`